Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.h

```diff
@@ -76,8 +67,11 @@ enum class NegotiatedLegacyTLS : bool;
 class LegacyCustomProtocolManager;
 class NetworkSessionCocoa;
 
-struct SessionWrapper : public CanMakeWeakPtr<SessionWrapper> {
-    void initialize(NSURLSessionConfiguration *, NetworkSessionCocoa&, WebCore::StoredCredentialsPolicy, NavigatingToAppBoundDomain);
+struct SessionWrapper : public CanMakeWeakPtr<SessionWrapper>, public CanMakeCheckedPtr<SessionWrapper> {
+    WTF_MAKE_STRUCT_FAST_ALLOCATED;
+    WTF_STRUCT_OVERRIDE_DELETE_FOR_CHECKED_PTR(SessionWrapper);
+
+    void initialize(NSURLSessionConfiguration*, NetworkSessionCocoa&, WebCore::StoredCredentialsPolicy, NavigatingToAppBoundDomain);
```

Comment:
Seems like when we call `initialize()` and `recreateSessionWithUpdatedProxyConfigurations()`, we should be holding a CheckedPtr / CheckedRef on the stack. However, I don't see it in this PR.

Reply:
FWIW, my plan was to do the deployment when CheckedPtr checker comes online but we could deploy them for these two member functions.